pages/info/about.md

```diff
@@ -69,7 +69,7 @@ Singularity does not utilize a daemon process to manage the containers. While da
 
 Additionally, securing a root owned daemon process which is designed to manipulate the host's environment becomes tricky. In currently implemented models, it is possible to grant permissions to users to control the daemon, or not. There is no sense of ACL's or access of what users can and can not do.
 
-While there are some other container implementations that do not leverage a daemon, they lack other features necessary to be considered as reasonable user facing solution without having root access. For example, there has been a standing unimplemented patch to RunC (already daemon-less) which allows for root-less usage (no root). But, user contexts are not maintained, and it will only work with chroot directories (instead of an image) where files must be owned and manipulated by the root user!
+There are several container implementations that do not leverage a daemon. In addition, they do not require the installation of setuid binaries to operate. An example of this is the runc project which has merged a long-standing patch. In addition, runc is the de-facto implementation of the Open Container Initiative's runtime specification, making it compatible with a wide variety of other systems. With the use of tools such as umoci, a user can create a chroot enviornment without privileges, removing previous concerns about requiring root privileges to set up a container's rootfs.
```

Comment:
the user will likely want links to these tools, and to open in a new page to not leave their current reading

Reply:
We would be doing a disservice to our users if we direct them to other projects which are not proven out on or designed for HPC computational systems, which is why most users are here.

Additionally, Singularity can create a chroot as a user without using root privileges. Granted pedantically, that would be using DockerHub so it is not truly creating a container from scratch as to build a container from scratch would require use of `fakeroot` and `debootstrap` and does not work with `yum` and some other distribution packaging systems (unless `umoci` has another method, if so I'd love to know more). The caveat is that the files created will be owned by the user, and not root. This maybe a red herring for a single user running locally, but it is not a method that can be used for creating reproducible container images.